usersync/usersyncers/syncer_test.go

```diff
@@ -109,6 +109,7 @@ func TestNewSyncerMap(t *testing.T) {
 		openrtb_ext.BidderTappx:        true,
 		openrtb_ext.BidderYeahmobi:     true,
 		openrtb_ext.BidderAdprime:      true,
+		openrtb_ext.BidderRevcontent:   true,
```

Comment:
Please add your adapter alphabetically to this list. Although to be fair, it seems to be in need of sorting.